addons/hr_payroll/static/src/js/payslip.js

```diff
@@ -0,0 +1,44 @@
+odoo.define('hr_payroll.payslip.tree', function (require) {
+"use strict";
+    var core = require('web.core');
+    var ListController = require('web.ListController');
+    var ListView = require('web.ListView');
+    var viewRegistry = require('web.view_registry');
+
+    var PayslipListController = ListController.extend({
+        buttons_template: 'PayslipListView.buttons',
```

Comment:
Instead of overriding this value, I would render the `PayslipListView.print_button` template and add it to this.$buttons in `renderButtons`